src/arrays.jl

```diff
@@ -1,6 +1,9 @@
 const Array_suppT = Union{Int64, CxxWrap.CxxLong,
                         Integer, Rational,
                         String, CxxWrap.StdString,
+                        Pair{CxxWrap.CxxLong,CxxWrap.CxxLong},
+                        Array{Pair{CxxWrap.CxxLong,CxxWrap.CxxLong}},
+                        Array{StdList{Pair{CxxWrap.CxxLong,CxxWrap.CxxLong}}},
```

Comment:
```suggestion
                        StdPair{CxxWrap.CxxLong,CxxWrap.CxxLong},
                        Array{StdPair{CxxWrap.CxxLong,CxxWrap.CxxLong}},
                        StdList{StdPair{CxxWrap.CxxLong,CxxWrap.CxxLong}},
```
these are the element types for the array, so we should have the list of pairs in here, instead of the `Array{StdList{Pair...` (to have the type required in #283)

also this should all probably be `StdPair` instead of Pair?